extensions/crc/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "crc",
-  "displayName": "Manages CRC",
-  "description": "Allow to start and stop CRC and use Podman",
+  "displayName": "CRC",
+  "description": "Allows the ability to start and stop CRC (CodeReady Containers) and use Podman Desktop to interact with it",
```

Comment:
AFAIK it's not called CodeReady containers anymore but just CRC
cc @gbraad ?

Reply:
Yeah, I thought so too but then all over the repo there's still references to CodeReady Containers. https://github.com/crc-org

Even I'm not sure what the name is now. Is it Red Hat OpenShift Local? Or CRC? or CodeReady Containers or CRC Runs Containers?